modin/experimental/backends/omnisci/query_compiler.py

```diff
@@ -293,20 +294,70 @@ def min(self, **kwargs):
         return self._agg("min", **kwargs)
 
     def sum(self, **kwargs):
+        min_count = kwargs.pop("min_count")
```

Comment:
this parameter is `sum` specific, we don't want it to be propagated into generic `sef._agg` method